src/linter.ts

```diff
@@ -19,13 +19,18 @@ const isAKnownException = (violation: IRuleResult, locations: IExceptionLocation
       continue;
     }
 
-    for (let i = 0; i < violation.path.length; i++) {
+    let arePathEquals: boolean = true;
+
+    for (let i = 0; arePathEquals && i < violation.path.length; i++) {
       if (location.path[i] !== violation.path[i]) {
+        arePathEquals = false;
```

Comment:
```suggestion
    let arePathEquals: boolean = true;

    for (let i = 0; arePathEquals && i < violation.path.length; i++) {
      if (location.path[i] !== violation.path[i]) {
        return false;
```

Reply:
> One suggestion that shouldn't result in a behavior change, I believe.

@P0lip I'm afraid it does. :-/ if we return `false` here, we won't process the potentially remaining locations. Let me see if I can refactor this a bit to get rid of those nested loops and make the intent clearer.

Reply:
>Let me see if I can refactor this a bit to get rid of those nested loops and make the intent clearer.

@P0lip Done. Do you like it better?

Reply:
@nulltoken it's better!